.gitignore

```diff
@@ -81,3 +81,6 @@ google-credentials.json
 web/runtime-shared/static
 
 etc/android-ndk-*
+
+# ignore server so `go build` doesn't break 'changes not in git' for lint
+server
```

Comment:
No workflow or shouldn't be building in this directory anyway. Are you adding this just to stop people with manual builds from accidentally ending up with something here? Or is there something that actually produces output here?

Reply:
Never got an answer on this one. If there IS something building here, that process should be fixed to not output here. If it's just to help guard against human uses, then it's fine to leave.

Reply:
this was to cope with the `go build ./web/cmd/server` in Dockerfile.cache on armhf

no longer needed with updated Dockerfile.cache, I deleted the .gitignore lines